server/routes/index.js

```diff
@@ -767,14 +767,15 @@ router
               {
                 $project: {
                   icon: 1,
+                  uid: 1,
                   _id: 0,
                 },
               },
             ],
-            as: 'icon',
+            as: 'configOwner',
```

Comment:
since this is already a `config` object I wonder if we can remove that from the name (since it is implied) and have a more descriptive name for this attribute like `ownerUser` if this is going to be a user object.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/196/commits/159c5b43ba6f258f9ca42cfd988c9f29017e7d09